Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -647,17 +657,17 @@ public class PBXProjGenerator {
         }
 
         for dependency in carthageDependencies {
-
+            
             let embed = dependency.embed ?? target.shouldEmbedCarthageDependencies
 
-            var platformPath = Path(carthageResolver.buildPath(for: target.platform))
+            var platformPath = Path(carthageResolver.buildPath(for: target.platform, isStatic: isStaticDependency(for: dependency)))
             var frameworkPath = platformPath + dependency.reference
             if frameworkPath.extension == nil {
                 frameworkPath = Path(frameworkPath.string + ".framework")
             }
             let fileReference = sourceGenerator.getFileReference(path: frameworkPath, inPath: platformPath)
 
-            if embed {
+            if embed && !isStaticDependency(for: dependency) {
```

Comment:
Static frameworks should not be embeded.

Reply:
We could flip this and check for the `if isStaticDependency(for: dependency) {` case first, then the else would be `else if embed`.

Reply:
Yep!